src/types/context.ts

```diff
@@ -0,0 +1,21 @@
+import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
+import { Octokit } from "@octokit/rest";
+import { AssistivePricingSettings } from "./plugin-input";
+import { createAdapters } from "../adapters";
+
+export type SupportedEvents = "issues.labeled" | "issues.unlabeled" | "label.edited" | "issue_comment.created";
```

Comment:
Maybe worth to use directly `Pick` from Github's type to avoid typos in here

Reply:
Maybe it'd be better, but it will fail anyway because of `T extends WebhookEventName`